jekyll/_cci2/arm-resources.md

```diff
@@ -33,7 +33,15 @@ key. CircleCI offers two Arm resources as part of the [`machine` executor]({{sit
 
 Which are available under these images:
 
-* `ubuntu-2004:202101-01` - most recent, recommended for all users
+* `ubuntu-2004:current` - most recent, recommended for all users
+* `ubuntu-2004:2022.04.1`
+* `ubuntu-2004:202201-02`
+* `ubuntu-2004:202201-01`
+* `ubuntu-2004:202111-02`
+* `ubuntu-2004:202111-01`
+* `ubuntu-2004:202107-01`
+* `ubuntu-2004:202104-01`
+* `ubuntu-2004:202101-01`
 * `ubuntu-2004:202011-01` - deprecated as of Feb 3, 2021
 
 As these are `machine` executor resources, each class is a dedicated VM that is created specifically for your job and subsequently taken down after the job has finished running.
```

Comment:
```suggestion
Because these are `machine` executor resources, each class is a dedicated VM that is created specifically for your job and subsequently taken down after the job has finished running.
```